pattern-2/tile/tile.yml

```diff
@@ -43,13 +43,38 @@ forms:
       default: true
     - name: com.microsoft.sqlserver.jdbc.SQLServerDriver
       label: com.microsoft.sqlserver.jdbc.SQLServerDriver
-  - name: carbon_db_query
+  - name: reg_db_query
     type: string
     label: Validation Query
-  - name: carbon_db_username
+  - name: reg_db_username
     type: string
     label: Username
-  - name: carbon_db_password
+  - name: reg_db_password
+    type: secret
+    label: Password
+- name: um_db
+  label: UM DB connection information
+  description: User Manager Database connection information
```

Comment:
The DB description should be **User Management Database connection information**